### PR TITLE
Update ms-ignite-2018.md

### PR DESCRIPTION
**Pull request by nikmd23**
I made lots of updates. We can discuss.

### CODE REVIEW

File: content/articles/ms-ignite-2018.md [hunk 1]
```diff
@@ -13,70 +13,42 @@ logo: /img/logo-sm.svg
...
-various programming languages and technologies closer.
+### OpenCensus spotlight
 
-### Keynote
-
-As part of a keynote Microsoft announced support of Virtual Nodes by
-Azure Kubernetes Service. Virtual Nodes allow to scale your app faster,
-in a matter of minutes. What's the best way to demonstrate this if not
-the Application Insights [Live
+During one of Ignite's keynotes, Microsoft announced support for Azure Kubernetes Service (AKS) Virtual Nodes. Virtual Nodes allow applications to scale more quickly, in a matter of mere minutes. The demonstration included a highlight of Azure Monitor for Application's [Live
```
Comment (SergeyKanzhelev) [inline content/articles/ms-ignite-2018.md]: "During one of Ignite's keynotes," weakens the message. Just say "During Ignite's keynote"?

File: content/articles/ms-ignite-2018.md [hunk 2]
```diff
@@ -13,70 +13,42 @@ logo: /img/logo-sm.svg
...
 
-Link to the [video](https://mediastream.microsoft.com/events/2018/1809/Ignite/player/tracks/track2.html?start=17300).
+The keen eyed among you may have noticed that this demo is powered by [OpenCensus](https://github.com/Azure-Samples/virtual-node-autoscale)!
 
-Looking at demo application you may see that it is powered by [Open
-Census](https://github.com/Azure-Samples/virtual-node-autoscale).
+![OpenCensus in Ignite Keynote](ms-ignite-2018-images/keynote-demo-app-uses-open-census.png)
 
-![](ms-ignite-2018-images/keynote-demo-app-uses-open-census.png)
+As mentioned in the demo itself, OpenCensus's vendor neutral instrumentation provides users the freedom to either leverage the features and power built into Azure Monitor, or to easily provide their own tooling within Azure. This is the benefit of OpenCensus and its community oriented approach.
```
Comment (SergeyKanzhelev) [inline content/articles/ms-ignite-2018.md]: OpenCensus wasn't mentioned in the demo. The openness of Azure was

File: content/articles/ms-ignite-2018.md [hunk 3]
```diff
@@ -13,70 +13,42 @@ logo: /img/logo-sm.svg
...
+OpenCensus simplifies the design of monitoring systems and opens the door for expanded polyglot scenarios and innovation. The power of these scenarios is of course amplified with support from the open source community, but also through efforts to bring unification and standardization across observability vendors.
 
-As a fundamental principle of Open Census we strive for consistency in
-data collection in SDKs thru specifications and test cases across
-languages. We build test cases for [propagation
-protocols](https://github.com/w3c/distributed-tracing) and data
-collection specifications.
+To that end, it is a fundamental principle of OpenCensus to strive for consistency across our libraries thru specifications. Microsoft has contributed to those efforts by contributing a test harness to help ensure compliance with the [Trace Context](https://github.com/w3c/distributed-tracing) propagation format, as well as Trace Context implementation to several OpenCensus language libraries. Microsoft will continue to champion open standards through participation in the W3C Distributed Tracing Working Group and W3C TPAC event in France this month.
```
Comment (SergeyKanzhelev) [inline content/articles/ms-ignite-2018.md:48]: There is more than W3C I alluded to. Like this one: https://github.com/census-instrumentation/opencensus-specs/pull/183

Comment (SergeyKanzhelev) [reply content/articles/ms-ignite-2018.md:48]: standard on data collection unification

File: content/articles/ms-ignite-2018.md (hunk 1 above)
Comment (nikmd23) [reply content/articles/ms-ignite-2018.md]: Fixed.

File: content/articles/ms-ignite-2018.md (hunk 2 above)
Comment (nikmd23) [reply content/articles/ms-ignite-2018.md]: Fixed.

File: content/articles/ms-ignite-2018.md (hunk 3 above)
Comment (nikmd23) [reply content/articles/ms-ignite-2018.md:48]: I've called #183 the "test harness". Would you like me to link to it, or are you saying I've called it the wrong thing?

Also, I'm not sure what *standard on data collection unification* means. Could you elaborate please?

**Review by SergeyKanzhelev — APPROVED** [review]

**Pull request merged**